### ISSUE TITLE
Support for newer scala versions

### ISSUE THREAD

**Issue by aliakhtar** [alexarchambault/plotly-scala#171]
It looks like this library was written for scala 2.11 and isn't compatible with newer versions (2.12, 2.13, etc). It'd be great if it could be updated.

**Comment by tmccarthy** [comment]
@aliakhtar The library has been published for Scala 2.12 and 2.13 since `0.7.1`.

**Comment by aliakhtar** [comment]
I couldn’t find it. Can you share the sbt line for adding it for 2.13
please?

**Comment by tmccarthy** [comment]
```scala
libraryDependencies += "org.plotly-scala" %% "plotly-render" % "0.7.2"
```

**Comment by aliakhtar** [comment]
🙏🏽

**Comment by alexarchambault** [comment]
Yes, plotly-scala is published for both scala `2.12` and `2.13` these days.
